winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -28,6 +28,7 @@
 #include <fcntl.h>
 #include <string.h>
 #include <ctype.h>
+#include <krb5/krb5.h>
```

Comment:
This include needs to be guarded for builds without working kerberos (looking at you apple)

Reply:
I removed the include altogether as it seems to be brought by the later `#include "krb5glue.h"`